src/improvements/tasks/MultisigTask.sol

```diff
@@ -813,7 +820,13 @@ abstract contract MultisigTask is Test, Script, ITask {
             if (
                 accountAccesses[i].account != address(addresses) && accountAccesses[i].account != address(vm)
                     && accountAccesses[i].accessor != address(addresses)
-                    && accountAccesses[i].kind == VmSafe.AccountAccessKind.Call && accountAccesses[i].accessor == multisig
+                    && (
+                        accountAccesses[i].kind == VmSafe.AccountAccessKind.Call
+                            || (
+                                accountAccesses[i].kind == VmSafe.AccountAccessKind.DelegateCall
+                                    && accountAccesses[i].depth == minDepth
+                            )
+                    ) && accountAccesses[i].accessor == multisig
```

Comment:
Can we make this `if` condition more readable by pulling checks into individual named bools and boiling this down to just `if (myFinalBool)`? This is pretty hard to parse